docs/blog/crucial-tests/index.md

```diff
@@ -394,6 +392,50 @@ beforeAll(() => {
 });
 ```
 
+## 📦 Test the package as a consumer
+
+a lot of times we don't test our packages as how our consumers will use them,
+this is a simplified example of how we could do that
+
+```js
+// global-setup.js
+
+// 1. Setup the in memory NPM registry
+await setupVerdaccio();
+
+// 2. Building our package 
+await execa('npm', ['run', 'build'], {
```

Comment:
execa is unneccessary concept that the reader should reason about

exec('command') is almost as easy and standard

Reply:
changed to exec